### PR TITLE
style(typings): Add tab panes prop definition

### PR DESCRIPTION
**Pull request by pierre-H**

### CODE REVIEW

File: src/modules/Tab/Tab.d.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export interface TabProps {
   onTabChange?: (event: React.MouseEvent<HTMLDivElement>, data: TabProps) => void;
 
   /** Shorthand props for the Menu. */
-  panes?: any;
+  panes?: Array<{menuItem: string; render: () => React.ReactNode;}>;
```
Comment (layershifter) [inline src/modules/Tab/Tab.d.ts]: Can we make it multiline?

```ts
panes?: Array<{
  menuItem: string;
  render: () => React.ReactNode;
}>;
```

**Review by layershifter — CHANGES_REQUESTED** [review]

File: src/modules/Tab/Tab.d.ts (hunk 1 above)
Comment (pierre-H) [reply src/modules/Tab/Tab.d.ts]: @layershifter done

**Review by layershifter — COMMENTED** [review]
@pierre-H Thanks again, LGTM

**Comment by layershifter** [comment]
I've relaxed type for changes from #1823.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.71.0`.

**Pull request merged**